apps/api/src/api/application-api.ts

```diff
@@ -105,14 +106,56 @@ export const getApplicationById = async ({ applicationId }: { applicationId: num
 
 	return result;
 };
-/**
- * Gets the total of how many applications are in each state type, including a TOTAL count.
- * @param userId - The ID of the current user.
- * @returns Success with the details of the application / Failure with Error.
- */
-export const getApplicationStateTotals = async ({ userId }: { userId: string }) => {
```

Comment:
This function was removed in application-api but is still used in `application-api.test`, has to be removed from that file as well

Reply:
Actually I think this may have been an incorrect merge; this function was added in #154 and should not be removed